// Wrapper pour capturer les erreurs asynchrones et les transmettre Ã  Express
module.exports = fn => {
    return (req, res, next) => {
      fn(req, res, next).catch(next);
    };
  };